src/js/components/agencyV2/statusOfFunds/IntroSection.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * IntroSection.jsx
+ * Created by Brian Petway 11/04/21
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from "react-redux";
+import GlossaryLink from "../../sharedComponents/GlossaryLink";
+
+const propTypes = {
+    fy: PropTypes.string
+};
+
+const IntroSection = ({ fy }) => {
+    const { name } = useSelector((state) => state.agencyV2.overview);
+    const agencyBudget = useSelector((state) => state.agencyV2.budgetaryResources?.[fy]?.agencyBudget) || '--';
+
+    return (
+        <div className="status-of-funds__intro-wrapper">
+            <div className="status-of-funds__intro-section-title">
+                <span>
```

Comment:
I think the `<span>` wrapper is unnecessary here? (Same on line 32)

Reply:
Correct, They have been removed.